cime_config/acme/machines/config_compilers.xml

```diff
@@ -790,6 +790,9 @@ for mct, etc.
 </compiler>
 
 <compiler OS="Linux" COMPILER="intel">
+  <CPPDEFS>
```

Comment:
Attempting to do that gets me a circular reference error for both of those variables